contributed_definitions/nyaml/NXpeak.yaml

```diff
@@ -1,136 +1,42 @@
 category: base
 doc: |
-  Description of peaks, their functional form or measured support.
+  Base class for peaks, their functional form, and measured support.
 symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
   n_support: |
     Number of support points
 type: group
 NXpeak(NXobject):
-  label:
+  label(NX_CHAR):
     doc: |
-      Human-readable identifier to specify which concept/entity
+      Human-readable label which specifies which concept/entity
       the peak represents/identifies.
-  peak_model:
+  peak_model(NX_CHAR):
     doc: |
       Is the peak described analytically via a functional form
       or is it empirically defined via measured/reported
       intensity/counts as a function of an independent variable.
       
-      If the functional form is not empirical or gaussian, users
+      If the functional form is not empirical or Gaussians, users
       should enter other for the peak_model and add relevant details
       in the NXcollection.
     enumeration: [empirical, gaussian, lorentzian, other]
```

Comment:
Can we add Voigt here to the enumeration? I.e., the voncolution of Gaussian and Lorentzian? I thinks it's quite common.

Also, Lorentzian and Gaussian should be upper case.

Reply:
I support this